app/Http/Controllers/Bookings/BookingRequestController.php

```diff
@@ -50,6 +52,7 @@ public function index(Request $request)
         return inertia('Admin/BookingRequests/Index', [
             'booking_requests' => BookingRequest::with('requester', 'reservations', $this->reservationRoom)->get(),
             'rooms' => Room::hideUserRestrictions($request->user())->with('availabilities', 'blackouts')->get(),
+            'paginator' => Room::hideUserRestrictions($request->user())->with('availabilities', 'blackouts')->paginate(self::ROOM_PAGINATOR_AMOUNT)
```

Comment:
Paginator holds everything you need for rooms. they both hold teh same data so this is making the same call twice. 
If you do not want to change to much i would suggest only returning the pagination data as rooms. Then in the vue, when you pass rooms into the table pass paginator.data since they are the same.

Reply:
I like this approach, will change it to that

Reply:
After a ton of testing and rewriting, a lot of tests depend on the passing of all the rooms. Also, the paginator.data only contains the rooms for that "page" and not all of them, so searches and filters would only apply to those. Same for the change requested below. @EvanDime

Reply:
I see. That's fine then.